user/tests/wiring/spi_master_slave/spi_master/test.mk

```diff
@@ -34,4 +34,9 @@ USE_CS_VAL=255
 endif
 
 CFLAGS += -DUSE_SPI=${USE_SPI_VAL} -DUSE_CS=${USE_CS_VAL}
-CXXFLAGS += -DUSE_SPI=${USE_SPI_VAL} -DUSE_CS=${USE_CS_VAL}
\ No newline at end of file
+CXXFLAGS += -DUSE_SPI=${USE_SPI_VAL} -DUSE_CS=${USE_CS_VAL}
+
+ifeq ("${SPI_SLEEP}","y")
```

Comment:
I would argue that testing this functionality should be enabled by default and there shouldn't be a conditional at all.